app/helpers/common_parameters_helper.rb

```diff
@@ -19,8 +19,8 @@ def parameter_value_content(id, value, options)
     content_tag(:span, options[:popover], :class => "input-group-addon") + lookup_key_field(id, value, options)
   end
 
-  def use_puppet_default_help link_title = nil, title = _("Use Puppet default")
-    popover(link_title, _("Do not send this parameter via the ENC.<br>Puppet will use the value defined in the manifest."), :title => title)
+  def skip_foreman_help link_title = nil, title = _("Skip prameter")
```

Comment:
Now that the title is more generic please change the places that use this helper and add a specific title to them.